internal/pkg/cli/pipeline_init.go

```diff
@@ -374,24 +464,37 @@ func (o *initPipelineOpts) selectURL() error {
 // bbhttps	https://huanjani@bitbucket.org/huanjani/aws-copilot-sample-service.git (fetch)
 // bbssh	ssh://git@bitbucket.org:teamsinspace/documentation-tests.git (fetch)
 
-// parseGitRemoteResults returns just the trimmed middle column (url) of the `git remote -v` results,
-// and skips urls from unsupported sources.
-func (o *initPipelineOpts) parseGitRemoteResult(s string) ([]string, error) {
-	var urls []string
-	urlSet := make(map[string]bool)
+// parseGitRemoteResults returns just the first (shortname) and second (url) columns of the `git remote -v` results as a map (url: name), and skips urls from unsupported sources.
+func (o *initPipelineOpts) parseGitRemoteResult(s string) map[string]string {
+	repos := make(map[string]string)
 	items := strings.Split(s, "\n")
 	for _, item := range items {
 		if !strings.Contains(item, githubURL) && !strings.Contains(item, ccIdentifier) && !strings.Contains(item, bbURL) {
 			continue
 		}
 		cols := strings.Split(item, "\t")
-		url := strings.TrimSpace(strings.TrimSuffix(strings.Split(cols[1], " ")[0], ".git"))
-		urlSet[url] = true
+		URL := strings.TrimSpace(strings.Split(cols[1], " ")[0])
+		repos[URL] = cols[0]
+	}
+	return repos
+}
+
+func (o *initPipelineOpts) parseGitBranchResults(s string) ([]string, error) {
+	var branches []string
+	branchSet := make(map[string]bool)
+	items := strings.Split(s, "\n")
+	for _, item := range items {
+		elements := strings.Split(item, "/")
```

Comment:
Does it work for Windows?

Reply:
We don't have pipelines for Windows, unfortunately.
But good question. A quick search tells me that git on Windows will still return slashes in the direction we're expecting here.

Reply:
Wait do you mean we don't have it for Windows container or running Copilot on Windows but for linux?

Reply:
I know pipeline doesn't support Windows environment but if they are just running Copilot on Windows machine that should be fine right?